mako.1.scd

```diff
@@ -107,11 +107,13 @@ dismissed with a click or via *makoctl*(1).
 
 	Default: #4C7899FF
 
-*--progress-color* _color_
+*--progress-color* [over|source] _color_
 	Set popup progress indicator color to _color_. See *COLOR* for more
-	information.
+	information. To draw the progess indicator on top of the background
+	color, use the *over* attribute. To replace the background color, use
+	the *source* attribute.
```

Comment:
Maybe note that `source` is mostly useful when the background is transparent, since it's not immediately obvious why the user should care about this distinction.